### PR TITLE
fixes #862: add procedure for launching WAR files with SB

### LINKED ISSUE #862: Spring Boot:  Document workflow for project packaged as a WAR file
Add example workflow for using a Spring Boot  project packaged as a WAR and which contains JSP pages and JSP Tage Library(JSTL). See [SB-171](https://github.com/fabric8-launch/appdev-documentation/issues/862).

### PR DESCRIPTION
**Pull request by inoxx03**
content not included in  any of the guides at the moment.

### CODE REVIEW

**Comment by inoxx03** [comment]
@lavernw1 FYI^^ :)

**Comment by lavernw1** [comment]
@inoxx03 thanks, taking a look now.  Laer I will add part 2 for the sample project skeleton as a follow-on and put in appendix.

**Comment by rhoads-zach** [comment]
@inoxx03 let me know when you want this merged.

**Comment by inoxx03** [comment]
@rhoads-zach Thank you. We haven't yet determined  where in the SB we want to include this. The content has been approved by Charles, so I think it's safe to merge now. It's just not included anywhere yet.
@lavernw1 Would you have a suggestion as to where in the Spring Boot runtime guide we should include this?

**Comment by lavernw1** [comment]
@rhoads-zach @inoxx03 The SB war files info is to be in an appendix. Additional information related to a sample starter project is to be added to go along with this content.  Plan was to add new appendix Spring Boot WAR file deployment after Appendix A.

**Comment by inoxx03** [comment]
@lavernw1 Thank you. Including this content as an appendix is IMHO a good idea. I'll make this Appendix B in the SB guide. Would you be OK with filing a follow-up PR to include the additional information? or would you prefer to add the to this PR? (Either way is fine for me:))

**Comment by lavernw1** [comment]
@inoxx03 thanks.  I'll file a  follow-on issue for the starter project.

**Comment by inoxx03** [comment]
@lavernw1 Thank you:) Also, I've updated the PR, so now the WAR file section appears as Appendix B of the Spring Boot runtime guide. WDYT?
@rhoads-zach removing the WIP form this PR, please go ahead and merge if this looks ok to you. Thanks :)

**Comment by inoxx03** [comment]
@lavernw1 Thank you for reviewing:) 
Context about using _Docker_ vs. _Docker-formatted_:
AFAIK, [Docker is now a trademark](https://www.docker.com/trademark-guidelines), so we should take care not to use the name too freely when we're talking strictly about the technology (as opposed to Docker as a company/product).
I'd be inclined to use _Docker-formatted image_ in Red Hat docs, because it refers to the technology we use, but not necessarily to the actual brand.

**Comment by lavernw1** [comment]
@inoxx03 yes we can use "Docker-formatted container image or just container image" to comply with the   Container Technology Language and Reference Guide

**Comment by inoxx03** [comment]
@lavernw1 Perfect! I see that the updated version of the _Container Technology Language and Reference Guide_ covers the naming convention around docker quite extensively:)

File: docs/topics/appendix-using-war-files-spring-boot.adoc [hunk 1]
```diff
@@ -0,0 +1,116 @@
+[[using-war-files-spring-boot]]
+= Building and Deploying {SpringBoot} Application WAR Files
+// Cnsider placing this inthe Spring Boot Runtime documentation instead?
+// You can Package and deploy your application as a war file
+[IMPORTANT]
+--
+Red Hat does not support packaging and deploying {SpringBoot} applications using WAR files in this release of {ProductShortName}.
```
Comment (rhoads-zach) [inline docs/topics/appendix-using-war-files-spring-boot.adoc]: @inoxx03 can you update this to `{ProductName}` or update document-attributes.adoc to re-enable `ProductShortName`

other than that, LGTM.

Comment (inoxx03) [reply docs/topics/appendix-using-war-files-spring-boot.adoc]: @rhoads-zach thanks for calling this out, I'll re-enable product short-name and update the PR.

**Comment by inoxx03** [comment]
@rhoads-zach Updated `document-attributes.adoc` to re-enable {ProductShortName}

**Pull request merged**